contrib/roles/windows/kubernetes/tasks/main.yml

```diff
@@ -38,7 +38,7 @@
 # and it's safer to have everything inside a powershell script to run.
 - name: Kubernetes | OVS Docker network setup
   include_tasks: ./setup_ovs_docker.yml
-  when: interface_name != "br-ex"
+  when: transparent_inspect.rc != 0
```

Comment:
This check should be improved. I think the following should work:
```
- name: Kubernetes | Check if system-id is set
  win_shell: ovs-vsctl get Open_vSwitch . external_ids:system-id 
  register: ovs_system_id
  ignore_errors: true
  changed_when: false

- name: Kubernetes | OVS Docker network setup
  include_tasks: ./setup_ovs_docker.yml
  when: ovs_system_id.rc != 0
```

Reply:
It still won't be sufficient to be honest.
I'll try to add some powershell commandlets to the OVS installer to improve the network creation/check/removal and send out a different PR for it.